src/command/init/helpers.rs

```diff
@@ -16,52 +16,99 @@ pub fn display_welcome_message() {
     println!();
 }
 
```

Comment:
I broke the logic between generating and displaying the successful project message to `stdout` to allow for easier testing of message content and more control of maintenance for display/generating behaviour 👍